_posts/en/newsletters/2020-06-17-newsletter.md

```diff
@@ -0,0 +1,193 @@
+---
+title: 'Bitcoin Optech Newsletter #102'
+permalink: /en/newsletters/2020/06/17/
+name: 2020-06-17-newsletter
+slug: 2020-06-17-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes the CoinPool payment pool proposal and
+the WabiSabi coordinated coinjoin protocol.  Also included are our
```

Comment:
Historical note: git grep shows a number of "CoinJoin" spellings in the 2018 newsletters.

Reply:
Yeah.  We mostly haven't gone back and applied our style guide rules retroactively.  I don't have a strong preference about whether or not we should do that---I like consistency, but I also consider newsletters older than a week or two to be static documents.